### PR TITLE
feat(dvcx): Add DVCX parameters to ConfigMap

### PR DESCRIPTION
**Pull request by mjasion**

### CODE REVIEW

File: charts/studio/templates/configmap-studio.yaml [hunk 1]
```diff
@@ -113,3 +113,5 @@ data:
   {{- else }}
   SOCIAL_AUTH_ALLOWED_REDIRECT_HOSTS: "studio-ui.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.studioUi.service.port }},studio-backend.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.studioBackend.service.port }}"
   {{- end }}
+
+  DQL_ROOT_DIR: {{ .Values.global.dql.rootDir | default "/tmp" | quote }}
```
Comment (omesser) [inline charts/studio/templates/configmap-studio.yaml:117]:
```suggestion
  DQL_ROOT_DIR: {{ .Values.global.dvcx.rootDir | default "/tmp" | quote }}
  DVCX_ROOT_DIR: {{ .Values.global.dvcx.rootDir | default "/tmp" | quote }}
```

Comment (omesser) [reply charts/studio/templates/configmap-studio.yaml:117]: + update  .Values.global.dql -> .Values.global.dvcx to new name

**Review by omesser — CHANGES_REQUESTED** [review]

File: charts/studio/values.yaml [hunk 2]
```diff
@@ -68,6 +68,11 @@ global:
     # -- Celery result URL
     resultBackend: ""
 
+  # -- Studio: Settings related to DVC^X
```
Comment (omesser) [inline charts/studio/values.yaml:71]:
```suggestion
  # -- Studio: Settings related to DVCX
```
just avoiding stuff that are hard to grep 😋

**Review by omesser — APPROVED** [review]

File: charts/studio/values.yaml [hunk 3]
```diff
@@ -68,6 +68,11 @@ global:
     # -- Celery result URL
     resultBackend: ""
 
+  # -- Studio: Settings related to DVCX
+  dvcx: {}
+    # -- Place where DVC^X will store its temporary files
```
Comment (jesper7) [inline charts/studio/values.yaml:73]:
```suggestion
    # -- Place where DVCX will store its temporary files
```

Same as Oded's comment from before

**Review by jesper7 — CHANGES_REQUESTED** [review]
LGTM. One minor comment

**Review by jesper7 — DISMISSED** [review]

**Pull request merged**